tests/core/test_upsert.py

```diff
@@ -352,3 +354,88 @@ def test_upsert_vector_with_pandas(index: Index):
     assert res[1].id == v2_id
     assert res[1].metadata is None
     assert res[1].vector == v2_values
+
+def test_upsert_data(index: Index):
```

Comment:
should we use different fixtures here and in test_query.py for tests which we use embedding models? (something like embedding_index after creating it in the conftest.py)

Reply:
what would be the use of having seperate fixtures for the embedding models, can't we use the existing ones?